diagnostic_updater/include/diagnostic_updater/update_functions.h

```diff
@@ -273,22 +270,31 @@ namespace diagnostic_updater
        * \brief Constructs the TimeStampStatus with the given parameters.
        */
 
-      TimeStampStatus(const TimeStampStatusParam &params) : 
-        DiagnosticTask("Timestamp Status"), 
+      TimeStampStatus(const TimeStampStatusParam &params, std::string name) :
+        DiagnosticTask(name),
         params_(params)
-    {
-      init();
-    }
+      {
+        init();
+      }
+
+      /**
+       * \brief Constructs the TimeStampStatus with the given parameters.
+       *        Uses a default diagnostic task name of "Timestamp Status".
+       */
+
+      TimeStampStatus(const TimeStampStatusParam &params) :
+        TimeStampStatus(params, "Timestamp Status") {}
```

Comment:
Same here, delegating constructors is a C++11 feature, and this package is still supporting Indigo which only supports C++03.